### PR TITLE
Update with id needed for Selenium test writing

### PR DESCRIPTION
**Pull request by Joy603**
Added IDs for verifying:

1. course code
2. sis id
3. course title
4. people link

### CODE REVIEW

File: course_info/templates/course_info/partials/details.html [hunk 1]
```diff
@@ -58,7 +58,7 @@ <h3 class="breadcrumbs">
 
             <li class="list-group-item">
                 <div class="row">
-                    <div class="col-md-2">
+                    <div id="course-title" class="col-md-2">
```
Comment (elliottyates) [inline course_info/templates/course_info/partials/details.html]: Please change to `input-course-title`

File: course_info/templates/course_info/partials/details.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h3 class="breadcrumbs">
 
             <li class="list-group-item">
                 <div class="row">
-                    <div class="col-md-2">
+                    <div id="sis-id" class="col-md-2">
```
Comment (elliottyates) [inline course_info/templates/course_info/partials/details.html]: Please change to `input-course-course_instance_id`

File: course_info/templates/course_info/partials/details.html [hunk 3]
```diff
@@ -18,7 +18,7 @@ <h3 class="breadcrumbs">
 
             <li class="list-group-item">
                 <div class="row">
-                    <div class="col-md-2">
+                    <div id="course-code" class="col-md-2">
```
Comment (elliottyates) [inline course_info/templates/course_info/partials/details.html]: Please change to `input-course-registrar_code_display`

**Comment by elliottyates** [comment]
To be forward-compatible with TLT-2376 I recommended changing some IDs. Otherwise tests will break in 2376. Otherwise :thu

**Comment by Joy603** [comment]
Funny.  The names got progressively longer.  I will update and merge them.  Thanks for the extra pair of eyes.

**Pull request merged**